app/internal/validation/check.go

```diff
@@ -75,27 +75,46 @@ func (l *List) Options(name, label string, value []string, checks ...OptionsChec
 	}
 }
 
-type SelectedCheck struct{}
+type SelectedCheck struct {
+	useCustomError bool
+}
+
+func (c SelectedCheck) CustomError() SelectedCheck {
+	c.useCustomError = true
+	return c
+}
```

Comment:
Nice compromise 👍

Reply:
Yeah, hopefully it doesn't end up spreading too far though